examples/imdb_bidirectional_lstm.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(1337)  # for reproducibility
+
+from keras.preprocessing import sequence
+from keras.optimizers import SGD, RMSprop, Adagrad
+from keras.utils import np_utils
+from keras.models import Sequential, Graph
+from keras.layers.core import Dense, Dropout
+from keras.layers.embeddings import Embedding
+from keras.layers.recurrent import LSTM, GRU
+from keras.datasets import imdb
+
+from sklearn.metrics import accuracy_score as accuracy
```

Comment:
Please do not import anything from sklearn, as it isn't supposed to be a Keras depency. We have our own accuracy utils in `keras/utils/np_utils.py`.